components/Workspaces/RepositoryStatCard.tsx

```diff
@@ -96,23 +96,27 @@ export const RepositoryStatCard = ({ stats, type, isLoading, hasError }: Reposit
             <span>{titles[type]}</span>
           </caption>
           <tbody className="grid grid-cols-3 items center">
-            {Object.entries(stats).map(([stat, value]) => (
-              <tr key={stat} className="flex flex-col">
-                <th scope="row" className="capitalize font-medium text-sm text-light-slate-11 text-left">
-                  {stat}
-                </th>
-                {stat === "health" ? (
-                  <td className="text-black semi-bold text-2xl">
-                    {Math.round(value)}
-                    <span className="text-xs">/10</span>
-                  </td>
-                ) : (
-                  <td className="semi-bold text-2xl" title={`${value}`}>
-                    {stat === "velocity" ? `${Math.round(value)}d` : humanizeNumber(value, "abbreviation")}
-                  </td>
-                )}
-              </tr>
-            ))}
+            {Object.entries(stats)
+              .filter(([stat]) => getStatPropertiesByType(type).includes(stat))
```

Comment:
Here's the fix. Now I filter on only the properties we want display. `activity_ratio` got introduced recently, but it's not something we want to display atm.

Reply:
Excellent: i'd imagine we'll introduce more and more of these and need time for them to bake / be evaluated, so 👍🏼